cmd/generator/config/config.go

```diff
@@ -733,4 +733,16 @@ func SetResourceConfigurations() {
 		},
 		UseAsync: true,
 	})
+	config.Store.SetForResource("azurerm_lb", config.Resource{
+		ExternalName: config.ExternalName{
```

Comment:
```suggestion
		Kind: "LoadBalancer",
		ExternalName: config.ExternalName{
```

FWIW, we opted for overriding the kind for such cryptic names.

Reply:
Thanks @muvaf. Renamed the type.